### PR TITLE
ini Message database table

### PR DESCRIPTION
**Pull request by K0725**

### CODE REVIEW

**Review by misslame — APPROVED** [review]
Perfect

**Review by PattyCakesxD — APPROVED** [review]
lgtm

**Comment by misslame** [comment]
You need to target `dev` branch

File: mesh/settings.py [hunk 1]
```diff
@@ -12,7 +12,10 @@
 
 from pathlib import Path
 import os
+from dotenv import load_dotenv
 
+
+load_dotenv()
```
Comment (peppacaiou) [inline mesh/settings.py:18]: this should not be needed, remove this, try again, and if it still doesn't work contact me

File: mesh/messages/models.py [hunk 2]
```diff
@@ -1,3 +1,9 @@
 from django.db import models
 
 # Create your models here.
+class Message(models.Model):
+    message_id = models.AutoField(primary_key=True)
+    from_account = models.ForeignKey(Account, on_delete=models.CASCADE, related_name='messages_from')
+    to_account = models.ForeignKey(Account, on_delete=models.CASCADE, related_name='messages_to')
+    message = models.TextField()
```
Comment (peppacaiou) [inline mesh/messages/models.py:8]: maybe include a max_length.

File: mesh/messages/models.py [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+    timestamp = models.DateTimeField(auto_now_add=True)
```
Comment (peppacaiou) [inline mesh/messages/models.py:9]: you're missing a string dunder method, I'm not sure how big of an impact it is, but it does make your admin updates look spicy and cool

File: mesh/settings.py (hunk 1 above)
Comment (K0725) [reply mesh/settings.py:18]: okok

**Comment by K0725** [comment]
> 

just updated my base branch!!

**Pull request merged**